_shared/project/tox.ini

```diff
@@ -31,6 +31,7 @@ setenv =
     dev: NEW_RELIC_ENVIRONMENT = {env:NEW_RELIC_ENVIRONMENT:dev}
     dev,tests,functests: PYTHONDEVMODE = {env:PYTHONDEVMODE:1}
     tests,functests: PYTEST_PLUGINS = tests.pytest_plugins.factory_boy
+    tests: COVERAGE_FILE = {env:COVERAGE_FILE:.coverage.{envname}}
```

Comment:
This is the same `COVERAGE_FILE` fix as in `ci.yml`, but this is for when running tox locally.

We can't use tox's `{envname}` for `COVERAGE_FILE` on CI because on CI `{envname}` evaluates to the same value for each version of Python being tested. That's why `ci.yml` has to override this by injecting its own `COVERAGE_FILE`